docs/advanced/payments/adyen.mdx

```diff
@@ -242,10 +234,22 @@ To allow loading Adyen related remote resources:
 -      connectSrc: [],
 +      connectSrc: ["*.adyen.com"],
       baseUri: []
-    }
+    },
++    reportOnlyDirectives: {
++      frameSrc: true,
++    }
```

Comment:
**issue:** this is a new feature from 2.19. Users with older versions must still be able to configure Adyen properly.

**suggestion:** add a comment here or above about the `frameSrc: ["*"]` for <2.19 so that it is clearer.

:warning: In parallel, document the `frameSrc = ["*"]` config in `main`. It will allow us to contact customers using Adyen to ensure they've authorized such iframes.

Reply:
reworded to document FC < 2.19